webapp/channels/src/sass/routes/_admin-console.scss

```
@@ -619,49 +625,67 @@
-            outline: none;
+        .input-wrapper {
+            display: flex;
+    
+            .filter {
+                flex-grow: 1;
+                padding: 6px 12px 6px 32px;
+                border: none;
+                border-radius: var(--radius-s);
+                background: rgba(var(--sys-sidebar-text-rgb), 0.08);
+                transition: background-color 0.2s ease-in-out;
+                color: var(--sys-sidebar-text);
```

Stylelint (17.14.0)

[error] Expected "color" to come before "transition" (order/properties-order)

(order/properties-order)